_posts/2018/2018-05-13-a-job-should-know-whether-to-run-itself.md

```diff
@@ -0,0 +1,92 @@
+---
+title: "A job should know whether to run itself"
+description: 'A little self.knowledge goes a long way.'
+layout: article
+category: ruby
+image:
+  base: '2018/a-job-should-know-whether-to-run-itself'
+  alt: 'A queue'
+  source: 'https://unsplash.com/photos/Nzb4LBsctyQ'
+  credit: 'Hal Gatewood'
+---
+
+Like other `ActiveWhatever` objects in Rails, a job also comes with a selection of callbacks that are run at different points in it's lifecycle.
```

Comment:
* Like other `Active`-prefixed objects in Rails, Active Job objects also come with a selection of...

Though, I'm also tempted to just have: Active Job objects come with a selection... as the starting.

Also "its", not "it's".